qiskit_runtime/vqe/vqe_program.py

```diff
@@ -0,0 +1,362 @@
+    def store_intermediate(self, store: bool) -> None:
+        """Whether or not to store intermediate information of the optimization."""
+        self._store_intermediate = store
+
+    @property
+    def callback(self) -> Callable:
+        """Returns the callback."""
+        return self._callback
+
+    @callback.setter
+    def callback(self, callback: Callable) -> None:
+        """Set the callback."""
+        self._callback = callback
+
+    def _wrap_vqe_callback(self) -> Optional[Callable]:
+        """ Wraps and returns the given callback to match the signature of the runtime callback. """
+
+        def wrapped_callback(*args):
+            # TODO: need to be completed
+            _, data = args  # first element is the job id
+            iteration_count = data[0]
+            params = data[1]
+            mean = data[2]
+            sigma = data[3]
+            return self._callback(iteration_count, params, mean, sigma)
+
+        # if callback is set, return wrapped callback, else return None
+        if self._callback:
+            return wrapped_callback
+        else:
+            return None
+
+    def compute_minimum_eigenvalue(self,
+                                   operator: OperatorBase,
+                                   aux_operators: Optional[List[Optional[OperatorBase]]] = None
+                                   ) -> MinimumEigensolverResult:
+        """Calls the VQE Runtime to approximate the ground state of the given operator.
+
+        Args:
+            operator: Qubit operator of the observable
+            aux_operators: Optional list of auxiliary operators to be evaluated with the
+                (approximate) eigenstate of the minimum eigenvalue main result and their expectation
+                values returned. For instance in chemistry these can be dipole operators, total
+                particle count operators so we can get values for these at the ground state.
+
+        Returns:
+            MinimumEigensolverResult
+
+        Raises:
+            ValueError: If the backend has not yet been set.
+
+        """
+        if self.backend is None:
+            raise ValueError('The backend has not been set.')
+
+        if self.provider is None:
+            raise ValueError('The provider has not been set.')
+
+        if not isinstance(operator, PauliSumOp):
+            try:
+                primitive = SparsePauliOp(operator.primitive)
+                operator = PauliSumOp(primitive, operator.coeff)
+            except Exception as exc:
+                raise ValueError(f'Invalid type of the operator {type(operator)} '
+                                 'must be PauliSumOp, or castable to one.') from exc
+
+        if self.initial_point is None:
+            initial_point = 'random'
+        else:
+            initial_point = self.initial_point
+
+        # combine the settings with the given operator to runtime inputs
+        # TODO: change 'random' to None in runtime inputs for initial state
```

Comment:
Did you mean to leave this TODO for later?

Reply:
hm looking at the VQE script I think this todo is already resolved